NitroxClient/MonoBehaviours/Multiplayer.cs

```diff
@@ -17,6 +17,7 @@ public class Multiplayer : MonoBehaviour
 
         public static Logic Logic;
         public static PacketSender PacketSender; //TODO: migrate logic out of and remove as member variable
+        public static Multiplayer main;
```

Comment:
Is there any other way to reference the Multiplayer component without this static reference? At the very least, I would recommend leveraging a static method to grab an instance.